src/api/app/models/event/comment.rb

```diff
@@ -20,6 +20,11 @@ def commenters
     return [] unless payload['commenters']
     User.find(payload['commenters'])
   end
+
+  def custom_headers
+    h = super
+    h['X-OBS-Request-Commenter'] = payload['commenter']
```

Comment:
You should use `originator.login` maybe, because `payload['commenter']` is the id of the user, so an integer...